astronomer_starship/starship_api.py

```diff
@@ -235,11 +235,30 @@ def airflow_version(self) -> str:
 
         **Response**:
         ```
-        OK
+        2.11.0+astro.1
         ```
         """
         return starship_route(get=starship_compat.get_airflow_version)
 
+    @expose("/starship_version", methods=["GET"])
```

Comment:
This could maybe be in the _other plugin_, too? That one kinda just deals with more generic stuff, where this one is more of the Airflow API stuff

Reply:
but `airflow_version` is indeed in `starship_api`, and it doesn't really matter much overall. Where you put it is fine

Reply:
ah, that other one. Didn't have it on the radar.

I'd say anything which is part of the documented Starship API and is intended for use also from outside the Starship UI (CLI tooling etc) may reside in **starship_api.py**.